### DISCUSSION TITLE
Applying promotion for each order by default from Admin

### DISCUSSION THREAD

**Discussion by ramkishore-yerubandi** [vendurehq/vendure#1465] [Q&A] [answered]
Is there any option from Admin to apply a promotion by default for each order that is getting created?

**Comment by michaelbromley** [comment, marked as answer]
You could e.g. set the promotion condition to "if order total is greater than 0", or even create a custom [promotion condition](https://www.vendure.io/docs/typescript-api/promotions/promotion-condition/#promotioncondition) that is just always true.

**Reply by ramkishore-yerubandi** [reply]
So if I don't define a coupon code and have a condition and action during the creation of Promotion, then this promotion will be applied on all Order..right? Am I understanding the concept right?

**Reply by ramkishore-yerubandi** [reply]
I mean. we don't need to call applyCoupon mutation on Order

**Reply by michaelbromley** [reply]
That's right, promotions do not require coupons. They need either a coupon _or_ at least 1 condition (or both).

**Reply by ramkishore-yerubandi** [reply]
thanks
